src/features/profile/HouseKeepingCard.js

```diff
@@ -16,6 +16,9 @@ import { CardContentSection } from "../../components"
 import { Link } from "../navigation"
 import { selectAllEntries } from "../entries"
 
+import {FATHOM_EXPORT} from "../../../src/constants"
+import { trackGoal } from "../../../src/features/tracking/index";
```

Comment:
**suggestion:** No need for the "/index" at the end.

**suggestion:** As little `../` as possible. `src` should never be a part of the import path. I think `../tracking` and `../../constants` would be correct

Reply:
../tracking and ../../constants #302
0ca89a1